apps/mail/app/(routes)/blog/[slug]/page.tsx

```
@@ -0,0 +1,153 @@
+                alt={post.title}
+                className="w-full h-full object-cover"
+              />
+              <div className="absolute inset-0 bg-gradient-to-t from-black/20 via-transparent to-transparent" />
+            </div>
+          )}
+
+          {/* Content */}
+          <article className="prose prose-invert prose-lg max-w-none mb-16">
+            <div 
+              className="text-[#a1a1aa] leading-relaxed text-lg [&>h1]:text-4xl [&>h1]:font-bold [&>h1]:text-white [&>h1]:mb-6 [&>h1]:mt-12 [&>h1]:leading-tight [&>h2]:text-2xl [&>h2]:font-bold [&>h2]:text-white [&>h2]:mb-5 [&>h2]:mt-10 [&>h2]:leading-tight [&>h3]:text-xl [&>h3]:font-semibold [&>h3]:text-white [&>h3]:mb-4 [&>h3]:mt-8 [&>h3]:leading-tight [&>p]:mb-6 [&>p]:leading-relaxed [&>p]:text-[#a1a1aa] [&>p]:text-lg [&>ul]:mb-6 [&>ul]:ml-6 [&>li]:mb-3 [&>li]:list-disc [&>li]:leading-relaxed [&>li]:text-[#a1a1aa] [&>li]:text-lg [&>strong]:text-white [&>strong]:font-semibold [&>em]:text-white/80 [&>a]:text-blue-400 [&>a]:hover:text-blue-300 [&>a]:underline [&>a]:underline-offset-2 [&>hr]:border-white/10 [&>hr]:my-12"
+              dangerouslySetInnerHTML={{ 
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)